components/openindiana/bhyve-fw/Makefile

```diff
@@ -0,0 +1,69 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2020 OmniOS Community Edition (OmniOSce) Association.
+#
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		uefi-edk2
+COMPONENT_VERSION=	20180309
+COMPONENT_FMRI=		system/bhyve/firmware
+COMPONENT_CLASSIFICATION=System/Core
+COMPONENT_SUMMARY=	UEFI-EDK2(+CSM) firmware for bhyve
+COMPONENT_SRC=		uefi-edk2-il-udk2014.sp1-1
+COMPONENT_ARCHIVE=	il-udk2014.sp1-1.tar.gz
+COMPONENT_ARCHIVE_HASH= \
+	sha256:d894d1ca61beeb0a1b76c2d6bbd137b02feb4783e4307f36e76efa29d415f6a8
+COMPONENT_ARCHIVE_URL= \
+	https://github.com/illumos/uefi-edk2/archive/$(COMPONENT_ARCHIVE)
+COMPONENT_PROJECT_URL=	https://github.com/illumos/uefi-edk2
+COMPONENT_LICENSE=	simplified-BSD
+
+include $(WS_MAKE_RULES)/prep.mk
+include $(WS_MAKE_RULES)/ips.mk
+
+COMPONENT_BUILD_ENV =
+COMPONENT_BUILD_ENV += GCCVER=4.4.4
```

Comment:
Why do we have to use GCC 4?

Reply:
Unfortunately this only builds with up to gcc 4.9, and SmartOS and OmniOS are building it with the illumos 4.4.4 compiler. It's building standalone UEFI firmware binaries so it's quite a delicate process.

The new branch which will be coming later in the year builds with more recent compilers.

Reply:
OK.